app/styles/atoms/input-container.less

```diff
@@ -45,10 +45,24 @@
     }
   }
 
-  &--errored {
```

Comment:
i don't see it reappearing anywhere ? if it's a typo, are you sure it was never used anywhere? Saw that you changed at least one test below for email-input. If there are others in the rest of the platform, basically all tests targeting this will start failing after other devs update the package.

So for BC either:
- leave this as is (when `error` keep using `errored` as BEM modifier)
- keep `--errored` AND add `--error` too and we will gradually update the tests to use the new class instead
- Go hunt for all of them and fix them too (not sure you want to do that ^^)